insights/specs/default.py

```diff
@@ -382,6 +383,29 @@ def du_dirs_list(broker):
     getconf_page_size = simple_command("/usr/bin/getconf PAGE_SIZE")
     getenforce = simple_command("/usr/sbin/getenforce")
     getsebool = simple_command("/usr/sbin/getsebool -a")
+
+    @datasource(Mount, [IsRhel6, IsRhel7, IsRhel8], HostContext)
+    def gfs2_mount_points(broker):
+        """
+        Function to search the output of ``mount`` to find all the gfs2 file
+        systems.
+        And only run the ``stat`` command on RHEL version that's less than
+        8.3. With 8.3 and later, the command ``blkid`` will also output the
+        block size info.
+
+        Returns:
+            list: a list of mount points of which the file system type is gfs2
+        """
+        gfs2_mount_points = []
+        if (broker.get(IsRhel6) or broker.get(IsRhel7) or
+                (broker.get(IsRhel8) and broker[IsRhel8].minor < 3)):
+            for mnt in broker[Mount]:
+                if mnt.mount_type == "gfs2":
+                    gfs2_mount_points.append(mnt.mount_point)
+        if gfs2_mount_points:
+            return gfs2_mount_points
+        raise SkipComponent
+    gfs2_file_system_block_size = foreach_execute(gfs2_mount_points, "/usr/bin/stat -fc %%s %s")
```

Comment:
@huali027 you also need to add a spec to `insights_archive.py` to enable access to the data in the archive.  I believe this will be the glob  `insights_commands\stat_-fc_s*`.

Reply:
@bfahr, since it is run by "foreach_execute", all the results are saved to "gfs2_file_system_block_size" and I tested it too, it works well. Do we still need to use glob to add a spec in insights_archive.py explicitly?

Reply:
@bfahr - this spec depends on a @datasource which is not supported by the old collection, which means it cannot be collected by the old collection. And the new collection will set up the relationship by generating a JSON file to the `meta_data` dir for all specs. Is it necessary to add it to the `insights_archive.py`?

Reply:
@huali027 @xiangce you are correct.  I had tested it incorrectly, and when I reran some additional tests I see that it works correctly as it.